src/main/java/com/flansmod/common/driveables/EntityVehicle.java

```diff
@@ -43,6 +43,7 @@
      */
     public int soundPosition;
     public int idlePosition;
+	public int exitPosition;
```

Comment:
Not sure where this is used, looks like it's just initialised and decremented, but the value isn't used or set anywhere.

Otherwise, good work :)